bitnami/sonarqube/templates/externaldb-secret.yaml

```
@@ -8,7 +8,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ printf "%s-externaldb" (include "common.names.fullname" .) }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ include "common.names.namespace" . }}
```

I would quote this given it's the only template without it

```suggestion
  namespace: {{ include "common.names.namespace" . | quote }}
```